src/redux/time/timeSlice.ts

```diff
@@ -14,6 +14,7 @@ export interface TimeState {
   nextDeltaTimeStep?: number;
   prevDeltaTimeStep?: number;
   deltaTimeSteps?: number[];
+  timeString?: string; // This is the raw OpenSpace time string we receive, can be used as backup if parsing fails.
```

Comment:
```suggestion
  timeString?: string; // This is the raw OpenSpace time string we receive, can be used as
                       // backup if parsing fails
 ```

Reply:
? We have various places where comments and strings overflow the allowed linewidth though...